modules/statics/src/base.ts

```diff
@@ -1438,7 +1438,7 @@ export enum KeyCurve {
  * This enum contains the base units for the coins that BitGo supports
  */
 export enum BaseUnit {
-  ATOM = 'uATOM',
+  ATOM = 'uatom',
```

Comment:
Will this break anything in WP or we use lowercase there ?

Reply:
We were using lowercase everywhere except here